Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
@clytwynec There isn't a standard .jshintrc file, unfortunately. Maybe the one in edx-ui-toolkit should be the canonical version, and then we could use Bower to install it into other repos:

https://github.com/edx/edx-ui-toolkit/blob/master/.jshintrc

@dsjen, would that be possible and/or is it a good idea? Obviously it only works for repos that consume the UI Toolkit, but the hope is that eventually the toolkit will benefit all front end repos.